### PR TITLE
Number with ordinal suffix

### PR DESCRIPTION
**Pull request by alexfilatov**
Hello! 

I just added function that adds ordinal suffix to the number, like 1st, 2nd, etc.
Please have a look and let me know whether this is ok to be added to the main repo.
Thanks!

### CODE REVIEW

File: lib/number/human.ex [hunk 1]
```diff
@@ -70,6 +70,21 @@ defmodule Number.Human do
     end
   end
 
+  @doc """
+  Adds ordinal suffix (st, nd, rd or th) for the number
+  """
+  def number_to_ordinal(number) when is_integer(number) do
+    sfx = ~w(th st nd rd th th th th th th)
+
+    (number |> Integer.to_string()) <> case number |> rem(100) do
```
Comment (danielberkompas) [inline lib/number/human.ex]: It isn't necessary to use the pipeline operator all the time; my personal rule is to only use it when I'm calling two or more functions in a row.

```elixir
Integer.to_string(number) <> case rem(number, 100) do
```

File: lib/number/human.ex [hunk 2]
```diff
@@ -70,6 +70,21 @@ defmodule Number.Human do
...
+      11 -> "th"
+      12 -> "th"
+      13 -> "th"
+      _ -> sfx |> Enum.at(number |> rem(10))
```
Comment (danielberkompas) [inline lib/number/human.ex]: Likewise here:

```elixir
_ -> Enum.at(sfx, rem(number, 10))
```

**Comment by alexfilatov** [comment]
thanks a lot for the notes, @danielberkompas !
I have made the changes, please have a look.

**Comment by danielberkompas** [comment]
Merged. Thanks! :star: :star: :star:

**Pull request merged**